src/lib/containers/GenericCard.tsx

```diff
@@ -83,7 +85,7 @@ export const getCutoff = (summary: string) => {
   return { previewText }
 }
 
-export const renderValueOrMultiValue = ({
+export const getValueOrMultiValue = ({
```

Comment:
And renamed this function (since we are not actually rendering anything in this function, but rather parsing out the values to render).  Also returns columnModelType (so caller can decide if it wants to use a specialized renderer for the returned value)